### PR TITLE
49594 ddl unit tests

### PR DESCRIPTION
**Pull request by kjduensing**
## Summary

Add unit tests for Decision Letter Download feature

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#49594

## Testing done

This is only unit tests

## What areas of the site does it impact?
CI

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [-]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/claims-status/containers/errorComponents/UnauthenticatedContent.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const UnauthenticatedContent = () => (
+  <div className="vads-u-text-align--center">
+    <h2 className="vads-u-font-size--h3">We can’t load this page</h2>
+    <div className="vads-u-font-size--lg">
+      Something went wrong on our end. Please double check the URL and make sure
+      you are signed in.
+    </div>
+  </div>
```
Comment (Mottie) [inline src/applications/claims-status/containers/errorComponents/UnauthenticatedContent.jsx:10]: I feel like all these error messages should be inside a `va-alert`. I haven't seen the design, but I feel like it makes sense.

**Review by Mottie — APPROVED** [review]

File: src/applications/claims-status/containers/errorComponents/UnauthenticatedContent.jsx (hunk 1 above)
Comment (jerekshoe) [reply src/applications/claims-status/containers/errorComponents/UnauthenticatedContent.jsx:10]: I personally agree with you, but this is the design that was approved:
https://www.sketch.com/s/8b025901-295d-4929-95cc-8dbeb28454b3/a/dlL1L52

Comment (jerekshoe) [reply src/applications/claims-status/containers/errorComponents/UnauthenticatedContent.jsx:10]: Screen Shot 2023-01-04 at 12 03 26 PM

**Comment by jerekshoe** [comment]
@kjduensing I'm thinking we should turn the YourClaimLetters container into a folder and put all of the associated components (ClaimLettersList, ClaimLettersListItem, and the error components) in it. We can take the code from `YourClaimLetters.jsx` and put it in the folder under an `index.jsx` file as well

**Pull request merged**